src/modeler/AutoRest.Swagger/Validation/Extensions/NextLinkPropertyMustExist.cs

```diff
@@ -54,7 +54,7 @@ public override bool IsValid(object pageable, RuleContext context, out object[]
         /// <remarks>
         /// This may contain placeholders '{0}' for parameterized messages.
         /// </remarks>
-        public override string MessageTemplate => "The property '{0}' specified by nextLinkName does not exist in the 200 response schema.";
+        public override string MessageTemplate => "The property '{0}' specified by nextLinkName does not exist in the 200 response schema. \nFor the method to return an IEnumerable, set nextLinkName to null. \nFor a method to return a chained object, set nextLinkName to appropriate field in the list object.";
```

Comment:
> For the method to return an IEnumerable

Careful with `IEnumerable`:
- this is Swagger validation, not C#
- people may think *"oh I want the signature to return IEnumerable! :)"* although they do have multiple pages. They may *reasonably* think that all pages are concat'ed into a single enumerable - something we should do anyways, so this message may even become deprecated.

I'd go with something like `If the API returns a single page`/`If the API does not return a chain of ...`